node_tree.go

```diff
@@ -258,15 +267,32 @@ func (t *NodeTree) Get(url string) (ok bool, n *Node, err error) {
 	return false, &Node{}, nil
 }
 
-// FullTextSearch uses a prebuild search index to perform a search
+// FullTextSearch uses a prebuilt search index to perform a search
 // over all possible attributes of each node.
 func (t *NodeTree) FullTextSearch(query string) ([]*Node, int, time.Duration) {
 	start := time.Now()
 
-	t.RLock()
-	defer t.RUnlock()
+	textQuery := bleve.NewMatchQuery(query)
+	textQuery.SetFuzziness(2)
```

Comment:
I think it would be more consequent to name the variable matchQuery, as the other query variables mirror the query method name, too.

Reply:
In Go it's okay to use shorter names for local variables. So in my opinion `mQuery` or even `mq`, would be okay in my book. As per:

> A rule of thumb:
The greater the distance between a name's declaration and its uses, 
the longer the name should be.

from https://talks.golang.org/2014/names.slide#4